diff --git a/configure.ac b/configure.ac
index 7c88f517..206c08cc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -66,7 +66,7 @@ GIO_REQUIRED=2.24.0
 GLIB_REQUIRED=2.50.0
 HARFBUZZ_REQUIRED=2.0.0
 LIBXML_REQUIRED=2.9.0
-PANGO_REQUIRED=1.48.11
+PANGO_REQUIRED=1.46.0
 
 dnl ===========================================================================
 
diff --git a/Cargo.toml b/Cargo.toml
index 180f5412..a2e508ba 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -12,11 +12,11 @@ cairo-svg = { version = "1.16", optional = true }
 gdk-pixbuf = { name = "gdk-pixbuf-2.0", version = "2.20" }
 gio = { name = "gio-2.0", version = "2.24" }
 glib = { name = "glib-2.0", version = "2.50" }
-pangocairo = "1.48"
+pangocairo = "1.46"
 
 [package.metadata.system-deps.'cfg(windows)']
 fontconfig = { version = "1.7", optional = true }
-pangoft2 = { version = "1.48", optional = true }
+pangoft2 = { version = "1.46", optional = true }
 harfbuzz = { version = "2.0", optional = true }
 freetype2 = { version = "20.0.14", optional = true }
 cairo = { version = "1.16", optional = true }
@@ -26,7 +26,7 @@ libxml2 = { name = "libxml-2.0", version = "2.9", optional = true  }
 
 [package.metadata.system-deps.'cfg(not(windows))']
 fontconfig = { version = "1.7" }
-pangoft2 = { version = "1.48" }
+pangoft2 = { version = "1.46" }
 cairo = "1.16"
 cairo-gobject = "1.16"
 cairo-png = "1.16"
